staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller_debug.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apiserver/pkg/server/mux"
+)
+
+func (cfgCtl *configController) Install(c *mux.PathRecorderMux) {
+	// TODO(yue9944882): handle "Accept" header properly
+	// debugging dumps a CSV content for three levels of granularity
+	// 1. row per priority-level
+	c.UnlistedHandleFunc("/debug/flowcontrol/dump_per_prioritylevel", cfgCtl.dumpPerPriorityLevel)
```

Comment:
How about `dump_priority_levels` instead of `dump_per_prioritylevel`, and so on?
We have been trying to converge on "API Priority and Fairness" as the user-facing name for this feature.  How about something like "apf" or "priority-and-fairness" instead of "flowcontrol"?

Reply:
renamed the paths as suggested